app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -59,8 +58,7 @@ def send_user_otp(method)
       current_user.create_direct_otp
 
       job = "#{method.capitalize}OtpSenderJob".constantize
-
-      job.perform_later(
+      job.perform_now(
```

Comment:
I think that `confirmation_context?` will tell us when we are confirming a number works for OTP